python/dgl/backend/backend.py

```diff
@@ -228,11 +228,45 @@ def context(input):
     pass
 
 def device_type(ctx):
-    """Return a str representing device type"""
+    """Return a str representing device type.
+    
+    Parameters
+    ----------
+    ctx : Device context object.
+        Device context.
+
+    Returns
+    -------
+    str
+    """
     pass
 
 def device_id(ctx):
-    """Return device index"""
+    """Return device index.
+    
```

Comment:
Can we have more docstring for the device index, for example -1 for cpu and 0 for cuda:0?

Reply:
For CPU, the device ID does not matter. For cuda, 0 means the first GPU, 1 means the second, etc. Will add more description.